src/components/shared/TinderCard/index.tsx

```diff
@@ -36,63 +36,80 @@ export interface TinderCardRef {
 
 interface Props<T> {
   testID?: string;
+  errorTestID?: string;
+  data?: T[];
+  renderCards?: (item: T) => ReactElement;
```

Comment:
If there's no `renderCards` props, we should be think about two case.
First, showing empty card state.
Second, `renderCards` function constraint.

Also, some of the props should have to `required` options for appropriate action.